verification/curator-service/ui/cypress/integration/components/SourceTableTest.spec.ts

```diff
@@ -12,7 +12,8 @@ describe('Sources table', function () {
         cy.get('button[title="Add"]').click();
         cy.get('input[placeholder="Name"]').clear().type('Example source');
         cy.get('input[placeholder="URL"]').clear().type('www.example.com');
-        cy.get('input[placeholder="Format"]').clear().type('JSON');
+        cy.get('div[data-testid="format-select"]').click();
+        cy.contains('li', 'JSON').click();
```

Comment:
This only really matters if there's another `li` with JSON in the DOM, but I think we can chain the `.contains[...]` off the previous `.click()`.

Reply:
unfortunately no, the new divs are not children of the format-select one :( it's this annoying "portal" elements thing that I can't seem to disable for selects.